src/SFA.DAS.Commitments.Application.UnitTests/Commands/UpdateApprenticeshipStatus/WhenStoppingAnAwaitingApprenticeship.cs

```diff
@@ -58,6 +59,9 @@ public override void SetUp()
             MockDataLockRepository.Setup(x => x.GetDataLocks(ExampleValidRequest.ApprenticeshipId, false))
                 .ReturnsAsync(new List<DataLockStatus>());
 
+            MockDataLockRepository.Setup(x => x.ResolveDataLock(It.IsAny<IEnumerable<long>>()))
+                .ReturnsAsync(1);
```

Comment:
The real ResolveDataLock always returns 0 (rather than 1), but as the return value is meaningless and never checked, it doesn't really matter what the mock returns.
We could tidy this up and change ResolveDataLock to just return a Task.